### DISCUSSION TITLE
[Docs] configuration/storage/migrations.md hints at schema downgrades but doesn't tell how to do them

### DISCUSSION THREAD

**Discussion by jinnatar** [authelia/authelia#5878] [Q&A] [answered]
### Version

v4.37.5

### Deployment Method

Docker

### Reverse Proxy

Traefik

### Reverse Proxy Version

N/A

### Description

`docs/content/en/configuration/storage/migrations.md` refers to sometimes needing a schema downgrade and provides a list of schema versions. It however does not document how to perform a schema downgrade.

### Reproduction

1. Visit https://www.authelia.com/configuration/storage/migrations/

### Expectations

Guidance is provided how to downgrade schema.

### Logs (Authelia)

```shell
N/A
```

### Documentation

https://www.authelia.com/configuration/storage/migrations/

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] This is a bug report and not a support request

- [X] I have read the security policy and this bug report is not a security issue or security related issue

- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration

- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide

- [X] I have checked for related proxy or application logs and included them if available

- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
Support requests should be made in the [Discussions](https://github.com/authelia/authelia/discussions). Issues are reserved for bug or feature requests with regards to the authelia binary. Thus I'll be converting this to a discussion.

**Reply by jinnatar** [reply]
I consider incomplete documentation to be a bug in documentation. But thanks for the cli reference, just linking to that from the main docs would be a decent fix.

**Reply by james-d-elliott** [reply]
```
Please only report bugs with Authelia itself. Issues which match one of the following criteria should not be reported here but should be a [discussion](https://github.com/authelia/authelia/discussions/new/choose) instead:

    Bugs with third-party software.
    Mistakes in the documentation.
    Potential bugs where it is not somewhat clear that it's a bug with Authelia itself.
```

**Comment by james-d-elliott** [comment, marked as answer]
See https://www.authelia.com/reference/cli/authelia/authelia_storage_migrate/

Would welcome a PR in improving this section of the docs.
